a3p-integration/proposals/a:vaults-auctions/test.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# Place here any test that should be executed using the executed proposal.
+# The effects of this step are not persisted in further proposal layers.
+
+# test more, in ways that change system state
```

Comment:
this comment doesn't make sense without `initial.test.js`.

Reply:
clarified.